libs/coin-modules/coin-stacks/src/utils.ts

```diff
@@ -1,5 +1,12 @@
 import { ResponseAddress } from "@zondax/ledger-stacks";
-import { isValidBase64, isValidHex } from "../filecoin/utils";
+
```

Comment:
perhaps you can create a folder logic with those utils ?

Reply:
Will be moved into their own package / folder after our merges ;)